README.md

```diff
@@ -1 +1,38 @@
-# elm-workshop
\ No newline at end of file
+# Codher Purely Functional Interfaces in Elm
+
+## Agenda
+
+- Getting started with Elm and Create Elm App
+- Intro into Functional Programming
+- Rendering HTML and CSS
+- Building interactive applications
+
+## Introduction
+
+Welcome to Codher's Elm Workshop!
+
+Functional Programming is fun and easy if you learn by doing.
```

Comment:
Maybe start with the benefits of functional programming (the immutable, pure and reactive paragraph) and then say it's easy and fun